src/main/java/com/mide/gangsaeng/board/BoardController.java

```diff
@@ -16,6 +19,8 @@
 @RequestMapping("/boards")
 public class BoardController {
 
+    public static final String DEFAULT_PAGE = "0";
```

Comment:
Why the page starts with 0?
Its confusing to client. Let's start with 1

And why it's type is string?

Reply:
I edited starts with 1. 
The reason its type is string is that the type of the `defaultValue` in `@RequestParam`.

Reply:
okay. it's fuckin spring annotation.

Reply:
> okay. it's fuckin spring annotation.

fuckin....?